### ISSUE TITLE
Table pagination with page size changer does not have aria-expanded attribute before interaction

### ISSUE THREAD

**Issue by kpustakhod** [ant-design/ant-design#32119]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://ant.design/components/table/#components-table-demo-ajax](https://ant.design/components/table/#components-table-demo-ajax)

### Steps to reproduce
go to https://ant.design/components/table/#components-table-demo-ajax, scroll to pagination in the bottom of the table, open devtools inspector, see that no aria-expanded attribute present before interaction.

https://user-images.githubusercontent.com/16884481/132863036-3159e8d9-e7de-42ef-a933-4cdda1a358a5.mov

### What is expected?
aria-expanded="false" expected to be present before interaction with this input.

### What is actually happening?
User that navigates with Voiceover and TAB to the page size changer will not understand that it is expandable.

| Environment | Info |
|---|---|
| antd | 4.16.13 |
| React | 17.0.0 |
| System | Mac |
| Browser | Chrome |

**Comment by yuzdeng** [comment]
Hi this issue is because the pagination components don't have defaultOpen value to the input component just like this issue I create: https://github.com/react-component/pagination/issues/345
Can you add my to your project then I can create pr for this issue?

**Comment by stefanonardo** [comment]
Same for `AutoComplete`.

**Comment by yoyo837** [comment]
Can this be closed safely?

**Comment by stefanonardo** [comment]
@yoyo837 I can still reproduce the issue
